c7n/resources/ec2.py

```diff
@@ -1531,12 +1532,15 @@ class Snapshot(BaseAction):
               - type: snapshot
                 copy-tags:
                   - Name
+                tags:
```

Comment:
could you add some content to the doc string, wrt to differentiating and identifying what these three different options that all have 'tag' in their name do.